app/models/gems_adoption.rb

```diff
@@ -1,10 +1,12 @@
 class GemsAdoption < ActiveRecord::Base
+  enum status: [ :open, :close ]
+
   belongs_to :user
   belongs_to :ruby_gem
   has_many :adoption_requests
   validates :ruby_gem, :user, presence: true
 
-  scope :search, -> (keyword) { includes(:ruby_gem).where("ruby_gems.name like ?", "%#{keyword}%").references(:ruby_gem) }
+  default_scope { where(status: 'open') }
```

Comment:
Normally the user of default scopes is not recommended since we end up forgetting that the result of `GemAdoption.all` will return results based on a scope.